docs/testcases/UI/Sample_Processing_Module/1589.md

```diff
@@ -14,39 +16,19 @@
 6. Select the file **`3-samples.csv`** and upload it
 7. Repeat steps 4-5, select the file **`3-samples-set.csv`** and upload it
 
-**Actions**:
-
-1. Open the **Library** page
-2. Open the folder created at step 3 of the preparations
-3. Click the **Metadata** item
-4. Click the "SampleSet <...>" item in the list
-5. Click the "Metadata" item in the breadcrumbs-control at the top of the page
-6. Click the "Sample <...>" item in the list
-7. Click the "**+ Add instance**" button
-8. Specify any valid "**Instance ID**" into the correspondding field
-9. Click the **Create** button
-10. Save the "**Created Date**" value in the row with ID specified at step 8
-11. Click the row with ID specified at step 8
-12. In the **Attributes** panel, click the "**+ Add**" button
-13. Specify any valid values for **Key** and **Value** fields (e.g. `key1`, `value1`)
-14. Click the **Add** button
-
-***
-
-**Expected result**:
-
-After step 4, 6:
-
-- the table contains the column "Created Date"
-- there are no empty cells in this column
-
-After step 9:
-
-- the table contains the row with ID specified at step 8
-- there is a non-empty value of the "Created Date" in this row
-
-After step 10, in the table:
-
-- there is a new column with the name equals to **Key** value specified at step 12
-- there is an only single column "Created Date"
-- "**Created Date**" value in the row with ID specified at step 8 equals to the saved at step 10
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Library** page | |
+| 2 | Open the folder created at step 3 of the preparations | |
+| 3 | Click the **Metadata** item | |
+| 4 | Click the "SampleSet <...>" item in the list | <li> the table contains the column "Created Date" <li> there are no empty cells in this column |
+| 5 | Click the "Metadata" item in the breadcrumbs-control at the top of the page | |
+| 6 | Click the "Sample <...>" item in the list | <li> the table contains the column "Created Date" <li> there are no empty cells in this column |
+| 7 | Click the "**+ Add instance**" button | |
+| 8 | Specify any valid "**Instance ID**" into the correspondding field | |
+| 9 | Click the **Create** button | <li> the table contains the row with ID specified at step 8 <li> there is a non-empty value of the "Created Date" in this row |
+| 10 | Save the "**Created Date**" value in the row with ID specified at step 8 | In the table: <br><li> there is a new column with the name equals to **Key** value specified at step 12 <li> there is an only single column "Created Date" <li> "**Created Date**" value in the row with ID specified at step 8 equals to the saved at step 10 |
+| 11 | Click the row with ID specified at step 8 | |
+| 12 | In the **Attributes** panel, click the "**+ Add**" button | |
+| 13 | Specify any valid values for **Key** and **Value** fields (e.g. `key1`, `value1`) | |
+| 14 | Click the **Add** button | |
```

Comment:
Move here the expected result from step 10

Reply:
Done